Packs/CybleEventsV2/README.md

```diff
@@ -0,0 +1,131 @@
+# Cyble Events
```

Comment:
Please update this file in order for it to match the structure in this [link](https://xsoar.pan.dev/docs/documentation/pack-docs#pack-readme) to xsoar pan dev website.